package.json

```diff
@@ -42,9 +42,8 @@
     "eslint": "eslint commands providers src --ext ts --cache",
     "eslint-fix": "npm run eslint -- --fix",
     "prepublishOnly": "npm run tsc",
-    "test": "npm run eslint",
-    "test-coverage": "npm run test-only -- --coverage",
-    "test-only": "jest",
+    "test": "jest",
+    "test-coverage": "jest --coverage",
```

Comment:
please keep the "test-only" script and update "test" to be: `npm run test-only && npm run eslint`